src/cmap/auth/mongodb_aws.ts

```diff
@@ -123,18 +136,15 @@ export class MongoDBAWS extends AuthProvider {
             path: '/',
             body
           },
-          {
-            accessKeyId: username,
-            secretAccessKey: password,
-            token
```

Comment:
This should've been `sessionToken`, I moved the whole object declaration to above.